sound/soc/sof/ipc4-priv.h

```diff
@@ -28,11 +28,13 @@ enum sof_ipc4_mtrace_type {
 /**
  * struct sof_ipc4_fw_module - IPC4 module info
  * @sof_man4_module: Module info
+ * @fw_mod_cfg: Pointer to the module config start of the module
  * @m_ida: Module instance identifier
  * @private: Module private data
  */
 struct sof_ipc4_fw_module {
 	struct sof_man4_module man4_module_entry;
+	const struct sof_man4_module_config *fw_mod_cfg;
```

Comment:
commit message: I couldn't follow the wording in quotes:


Save a pointer to the firmware module configuration area in the manifest
"for each module which has at least one for later use"

at least one what?

Reply:
Sure, I reword it: If the module have mod_cfg we save it for later use. The has or not has does not give any added information, we save the mod_cfg of the module, if it has, we have a pointer if it does not have then we have a NULL.